modules/digitaluniverse/shaders/billboard_gs.glsl

```diff
@@ -101,91 +85,94 @@ void main() {
     } else if (pos.w == 6.f) {
         unit = 306391534.73091 * PARSEC;
```

Comment:
I know it's not your code, but if you know, could you put a comment where that number comes from?

Reply:
Done and shamefully it is my code. :-D